src/main/java/org/folio/consortia/service/HttpRequestServiceImpl.java

```diff
@@ -35,7 +35,11 @@ public PublicationHttpResponse performRequest(String url, HttpMethod httpMethod,
     var absUrl = folioExecutionContext.getOkapiUrl() + url;
     log.debug("performRequest:: folio context header TENANT = {}" , folioExecutionContext.getOkapiHeaders().get(XOkapiHeaders.TENANT).iterator().next());
 
-    var responseEntity = restTemplate.exchange(absUrl, httpMethod, httpEntity, Object.class);
+    var responseEntity = switch (httpMethod.toString()) {
+      case "GET" -> restTemplate.exchange(absUrl, httpMethod, httpEntity, Object.class);
+      case "POST", "PUT", "DELETE" -> restTemplate.exchange(absUrl, httpMethod, httpEntity, String.class);
+      default -> throw new IllegalStateException("Unexpected value: " + httpMethod);
```

Comment:
These changes introduced to fix these below problems:
1) Requested DELETE Method:
![image](https://github.com/folio-org/mod-consortia/assets/113523904/e456b5fb-f58c-4aea-8d6a-0d6717764fe5)

2. While saving POST request which has two tenants 
```
Current thread: http-nio-8081-exec-4
19:29:58 ConsortiaAsync-2 [] [diku] [5c780098-751b-4477-9b61-0f75bc452a73] [mod-consortia] DEBUG cutionContextManager FolioExecutionContext removed;
Current thread: ConsortiaAsync-2
19:29:58 ConsortiaAsync-2 [] [diku] [5c780098-751b-4477-9b61-0f75bc452a73] [mod-consortia] ERROR blicationServiceImpl savePublicationTenantRequest:: error saving publication tenant request 541367d0-4b9f-44b3-adf8-44fe93f77f1d
org.springframework.orm.jpa.JpaObjectRetrievalFailureException: Unable to find org.folio.consortia.domain.entity.PublicationStatusEntity with id 2cd2169a-aeb1-43d1-9788-503c2f00b993
	at org.springframework.orm.jpa.EntityManagerFactoryUtils.convertJpaAccessExceptionIfPossible(EntityManagerFactoryUtils.java:376) ~[spring-orm-6.0.8.jar:6.0.8]
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:234) ~[spring-orm-6.0.8.jar:6.0.8]
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.translateExceptionIfPossible(AbstractEntityManagerFactoryBean.java:550) ~[spring-orm-6.0.8.jar:6.0.8]
```